### ISSUE TITLE
Getting node 12 deprecation error from github workflows runner

### ISSUE THREAD

**Issue by wickkidd** [gradle/wrapper-validation-action#66]
`Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: gradle/wrapper-validation-action`

I am using v1 `- uses: gradle/wrapper-validation-action@v1` and that tag seems to include the bump to node 16 from last year.

**Comment by wickkidd** [comment]
Ah ok. Looks like we have until summer '23 before they pull the plug on node 12.

https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/

Should I make a request for this or is it already planned to happen by then?

**Comment by qhy040404** [comment]
We have the upgrade commit, but we don't have the release. 
We can use `gradle/wrapper-validation-action@master` at the moment.

**Comment by JLLeitschuh** [comment]
Unfortunately, I no longer work for Gradle, so I don't have the ability to do so.

CC: @big-guy, @eskatos , @pioterj

**Comment by TWiStErRob** [comment]
PR cross ref: https://github.com/gradle/wrapper-validation-action/pull/53

@big-guy, @eskatos, @pioterj once you release this, please also give some love to https://github.com/gradle/gradle-build-action/issues/461.

Feels like GitHub deprecated the whole world :)

**Comment by TWiStErRob** [comment]
@bigdaz could you please release this one too?

**Comment by sciencewhiz** [comment]
I believe actions/core will need to be updated too

**Comment by TWiStErRob** [comment]
@sciencewhiz what's the need for that? There's no warnings about set-output/set-state here.
Here's an example of a most recent execution:
https://github.com/detekt/detekt/actions/runs/3306311082

**Comment by bigdaz** [comment]
I've updated a couple of dependencies (that had security vulnerabilities), and released the latest code as `v1.0.5`. I have not yet updated the `v1` tag to point to the new release, because I'm a little concerned that I may have broken things.

Can a few folks watching this issue please test with `gradle/wrapper-validation-action@v1.0.5` and report back if it's working (or not)? Once confirmed, I'll complete the release process by force-pushing the `v1` tag.

**Comment by NotMyFault** [comment]
> Can a few folks watching this issue please test with gradle/wrapper-validation-action@v1.0.5 and report back if it's working (or not)? Once confirmed, I'll complete the release process by force-pushing the v1 tag.

Hey,

thanks for the update. I can assure, that v1.0.5 works fine in my environments. This applies to highlighting wrong hashes:
![Screenshot 2022-10-24 at 18 46 49](https://user-images.githubusercontent.com/13383509/197580820-6fdf3f79-b78c-49d1-88e2-90ac269c4079.png)
and passing on correct ones:
![Screenshot 2022-10-24 at 18 47 12](https://user-images.githubusercontent.com/13383509/197580864-eaa39c5d-1dcf-4e2d-abcc-37eedcadf8bc.png)

Thanks 👏🏻

**Comment by TWiStErRob** [comment]
I'm very confused about the `v1.0.5` tag, it looks like it's a separate branch, whose only difference to `master` is an added `node_modules` folder, which is almost never version controlled, it's like pushing `~/.gradle/caches/files-2` to a repo. What happened there?

**Comment by TWiStErRob** [comment]
Got a success too with the [new version](https://github.com/TWiStErRob/detekt/pull/1/files):
https://github.com/TWiStErRob/detekt/actions/runs/3314729223/jobs/5474385497#step:3:7

**Comment by bigdaz** [comment]
@TWiStErRob I simply followed the [documented release process](https://github.com/gradle/wrapper-validation-action/blob/master/RELEASING.md).  I agree that this is odd, and it's not how the `gradle/gradle-build-action` repository/releases are structured.

But since this is my first ever involvement with this repo, I decided to keep my changes to a minimum.

Thanks for confirming the new version works as expected. I'll force-push `v1` to point to `v1.0.5` shortly.
